Changelog.md

```diff
@@ -42,6 +42,7 @@ Breaking Changes:
  * Type Checker: Disallow uninitialized storage variables. This was already the case in the experimental 0.5.0 mode.
  * Type Checker: Only accept a single ``bytes`` type for ``.call()`` (and family), ``keccak256()``, ``sha256()`` and ``ripemd160()``.
  * Type Checker: Fallback function must be external. This was already the case in the experimental 0.5.0 mode.
+ * Type Checker: Interface functions must be declared external.
```

Comment:
``This was already the case in the experimental 0.5.0 mode.`` :).

Reply:
Exactly. I assumed that Breaking-Changes list is for non-experimental-feature-users. But I can remove it of course.

Reply:
I meant to add the phrase ``This was already the case in the experimental 0.5.0 mode.`` to the end of the changelog entry (sorry for being word-lazy :)) - we did that for all the other breaking changes that were already there in experimental mode, see e.g. the entry above.

Reply:
At the end we might group them somehow.